packages/web/src/components/gcds-fieldset/gcds-fieldset.css

```diff
@@ -30,6 +34,12 @@
 
       .legend__required {
         margin: var(--gcds-fieldset-legend-required-margin) !important;
+        font: var(--gcds-text-size-body-desktop); /* --gcds-fieldset-legend-required-font-desktop */
```

Comment:
It seems that we are only changing the font-weight here which means we don't have to add desktop and mobile styles. Can we either add a new token for this or use the new fieldset required label token to make this font change instead of the general text token? Keeps it a little bit more semantic.

Reply:
Sorry I realized I didn't upload all the changes needed to the fieldset CSS for the required label change.